notebooks/6_Putting_it_All_Together.ipynb

```diff
@@ -0,0 +1,1002 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "kpcovr_hypers = dict(mixing=0.5, n_components=5)\n",
+    "\n",
+    "kpcovr = KPCovR(kernel=\"linear\", center=True, **kpcovr_hypers)\n",
+    "kpcovr.fit(\n",
+    "    Phi_train, Y_train, \n",
+    "    Yhat=Yp_train # This will throw an error if Y has not been estimated.\n",
```

Comment:
not sure what the comment means. How can I know/make sure Y has been estimated, and what does "estimate Y" means in this context? 

If this is all material covered in previous notebooks, then my question becomes how much do we want to couple this notebook to previous ones?

Reply:
Addressed in most recent commit!